packages/adapter-node/README.md

```diff
@@ -40,11 +40,44 @@ Enables precompressing using gzip and brotli for assets and prerendered pages. I
 By default, the server will accept connections on `0.0.0.0` using port 3000. These can be customised with the `PORT` and `HOST` environment variables:
 
 ```
-HOST=127.0.0.1 PORT=4000 node build
+HOST=127.0.0.1 PORT=4000 node build/server
 ```
 
 You can specify different environment variables if necessary using the `env` option.
 
+## Middleware
+
+Adapter generates a [connect](https://github.com/senchalabs/connect) compatible middleware `(req, res, next) => {}`.
+
+You can have a look at `build/server.js` to study a simple reference implementation
+on how to use this with a plain node http server.
+
+You can of course your favorite server framework to combine with other and/or custom middleware, e.g. [express](https://github.com/expressjs/expressjs.com).
+
+```
```

Comment:
Other questions the docs should answer: Where do you put this file? Is there some option you need to specify to make it use this vs the default?

Reply:
Not sure what you mean. Using it this way is kind of advanced stuff for experienced node users. So I am not sure how to verbose to document. I'm not a native speaker, so feel free to come up with better documentations.

Reply:
I mean if you want to create your own server using the code below does it matter what the file path or name is?